.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,61 @@
+      
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v4
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)